manifest.md

```diff
@@ -246,39 +215,22 @@ image. It's the direct replacement for the schema-1 manifest.
 
 # Backward compatibility
 
-The registry will continue to accept uploads of manifests in both the old and
-new formats.
-
-When pushing images, clients which support the new manifest format should first
-construct a manifest in the new format. If uploading this manifest fails,
-presumably because the registry only supports the old format, the client may
-fall back to uploading a manifest in the old format.
-
-When pulling images, clients indicate support for this new version of the
-manifest format by sending the
-`application/vnd.docker.distribution.manifest.v2+json` and
-`application/vnd.docker.distribution.manifest.list.v2+json` media types in an
-`Accept` header when making a request to the `manifests` endpoint. Updated
-clients should check the `Content-Type` header to see whether the manifest
-returned from the endpoint is in the old format, or is an image manifest or
-manifest list in the new format.
-
-If the manifest being requested uses the new format, and the appropriate media
-type is not present in an `Accept` header, the registry will assume that the
-client cannot handle the manifest as-is, and rewrite it on the fly into the old
-format. If the object that would otherwise be returned is a manifest list, the
-registry will look up the appropriate manifest for the amd64 platform and
-linux OS, rewrite that manifest into the old format if necessary, and return
-the result to the client. If no suitable manifest is found in the manifest
-list, the registry will return a 404 error.
-
-One of the challenges in rewriting manifests to the old format is that the old
-format involves an image configuration for each layer in the manifest, but the
-new format only provides one image configuration. To work around this, the
-registry will create synthetic image configurations for all layers except the
-top layer. These image configurations will not result in runnable images on
-their own, but only serve to fill in the parent chain in a compatible way.
-The IDs in these synthetic configurations will be derived from hashes of their
-respective blobs. The registry will create these configurations and their IDs
-using the same scheme as Docker 1.10 when it creates a legacy manifest to push
-to a registry which doesn't support the new format.
+The registry will continue to accept uploads of manifests in both the old and new formats.
```

Comment:
We really need to add distribution-agnostic fallback. A mimetype table and optional distribution layer aren't really sufficient for describing fallback that will work across transports. Otherwise, we risk losing compatibility across transports.

Perhaps, this section should describe the structure of the table.